Source/WebCore/platform/graphics/FontDescription.cpp

```diff
@@ -68,6 +68,7 @@ FontDescription::FontDescription()
     , m_fontStyleAxis(enumToUnderlyingType(FontStyleAxis::slnt))
     , m_shouldAllowUserInstalledFonts(enumToUnderlyingType(AllowUserInstalledFonts::No))
     , m_shouldDisableLigaturesForSpacing(false)
+    , m_enableEvaluationTimeZoom(false)
```

Comment:
`enable` in a boolean is weird (it's an action). We could just use `bool m_evaluationTimeZoom` and the setter would be `setEvaluationTimeZoom(true/false)`